Makefile

```diff
@@ -52,7 +54,7 @@ docker-build:
 	docker build -t $(IMAGE_NAME) .
 
 docker-run:
-	docker run -d --name $(CONTAINER_NAME) -p 8000:8000 $(IMAGE_NAME)
+	docker run -d --name $(CONTAINER_NAME) -p $(CONTAINER_PORT):8000 -e host=$(CONTAINER_ADDRESS) $(IMAGE_NAME)
```

Comment:
@rkorsak Is the port backwards? Aren't we always expecting container to have port 8000 but the exterior port might change?

Port exposed by container : **Port we connect to**

Reply:
It's actually the other way around!  [The first half of the port mapping defines how the port is exposed outside of the container](https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p---expose).

I did double check that this works by mapping to a non-8000 port and hitting it with a ping. 👍